charts/stable/games-on-whales/README_CHANGELOG.md.gotmpl

```diff
@@ -9,6 +9,23 @@ All notable changes to this application Helm chart will be documented in this fi
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [1.1.1]
+
+#### Added
+
+- N/A
+
+#### Changed
+
+- Unlimit /dev/shm for retroarch - needed for [some emulators](https://github.com/libretro/dolphin/issues/222)
+- correct sound devices group in retroarch
+
+#### Removed
+
+- N/A
+
+[1.1.1]: #111
```

Comment:
The convention is to collect the version 'links' at the bottom of the file. Do you have a specific reasoning for keeping them mixed in with the changelog?

Reply:
too much copy&paste from the changelog template...

Anyway, since we combine multiple charts in the same repository, the bottom section of the changelog does not make much sense in this repo. Should we remove it from the template?

Reply:
I believe the anchor links per version won't work if we don't include these somewhere, that's why we collect them in the bottom section. But we're rethinking our changelog approach, so this is something we'll take in to account for sure!